### PR TITLE
Fix ordering of articles with interstitial markers ("AAAAA")

### PR DESCRIPTION
**Pull request by ronnix**
https://fr.wikipedia.org/wiki/Association_amicale_des_amateurs_d%27andouillette_authentique

### CODE REVIEW

**Comment by ronnix** [comment]
Should fix https://rollbar.com/zam/zam/items/51/ and https://rollbar.com/zam/zam/items/52/

File: repondeur/tests/models/test_article.py [hunk 1]
```diff
@@ -77,6 +88,72 @@ def test_types(self, lecture_an):
...
+                    "17 bis BB",
+                    "17 bis B",
+                    "17 bis C",
+                    "17 bis",
+                    "17 ter",
+                ]
+            )
+        ),
+    )
+    def test_andouillette(self, lecture_an, before, after):
```
Comment (davidbgk) [inline repondeur/tests/models/test_article.py]: ❤️ can't wait for that one to fail, it deserves a dedicated pytest plugin!

Comment (ronnix) [reply repondeur/tests/models/test_article.py]: 😋

**Comment by boogheta** [comment]
You should really explore first the pieces of code of tlfp ;)
https://github.com/regardscitoyens/the-law-factory-parser/blob/master/tlfp/tools/sort_articles.py

**Pull request merged**